.gitignore

```diff
@@ -5,6 +5,8 @@ phpunit.xml
 sample/HelloWorld/vendor/
 sample/HelloWorld/cache/
 sample/HelloWorld/composer.lock
+sample/HelloWorld/composer.phar
```

Comment:
why are you re-downloading composer in each sample? i don't think thats what we should suggest for users.